src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -43,6 +44,7 @@ public class TelemetryManagerTests
         private Mock<ILogger> loggerMock;
         private Mock<ITelemetryTimer> telemetryTimerMock;
         private Mock<IKnownUIContexts> knownUIContexts;
+        private ICurrentTimeProvider currentTimeProvider = DefaultCurrentTimeProvider.Instance;
```

Comment:
The current TelemetryManager tests work ok without us needing to inject specific date times so I didn't try to change them to use a mock instead.

However, I will the mock when adding new tests to fix the "increment days of use" bug.